include/net/fota_download.h

```diff
@@ -31,7 +31,8 @@ extern "C" {
  */
 enum fota_download_evt_id {
 	/** FOTA download finished. */
-	FOTA_DOWNLOAD_EVT_FINISHED,
+	FOTA_DOWNLOAD_EVT_MCUBOOT_FINISHED,
```

Comment:
Should we use "BL1" here instead of mcuboot? In the future we might use something else as the second stage bootloader. Maybe even just call ut "BOOTLOADER"

Reply:
event removed